src/css/in-page.css

```diff
@@ -233,6 +288,54 @@
   display: inline-block;
 }
 
+/* Premium: Filter by Active Site */
+.fx-relay-menu-filter-active-site {
+  display: none;
+  overflow: hidden;
+}
+
+.fx-relay-menu-filter-active-site.is-visible {
+  display: flex;
+  width: 100%;
+  align-items: stretch;
+}
+
+.fx-relay-menu-filter-active-site-button {
+  --fxRelayMenuFilterActiveSiteButtonBorderHeight: 3px;
+  width: 50%;
+  background: transparent;
+  appearance: none;
+  outline: none;
+  border: none;
+  border-bottom: var(--fxRelayMenuFilterActiveSiteButtonBorderHeight) solid transparent;
+  color: var(--colorGrey50);
+  text-align: center;
+  padding: var(--spacingMd) 0;
+  position: relative;
+  font-weight: 500;
+  cursor: pointer;
+}
+
+.fx-relay-menu-filter-active-site-button + .fx-relay-menu-filter-active-site-button::after {
+  content: "";
+  top: 0;
+  left: 0;
+  width: 1px;
+  height: calc(100% + var(--fxRelayMenuFilterActiveSiteButtonBorderHeight));
+  z-index: 1;
```

Comment:
Likewise: z-index comment?

Reply:
Comment added in 80402c57eaa99b1b1617691f015a18f1ed6fda02